.github/workflows/sbom.yaml

```diff
@@ -90,12 +90,12 @@ jobs:
       - name: Generate and Validate
         run: |
           img=$(go run ./ build ./)
-          go run ./ deps $img --sbom=spdx > sbom.txt
+          go run ./ deps $img --sbom=spdx > sbom.json
 
-          java -jar ./spdx-tools-2.2.7-jar-with-dependencies.jar Verify sbom.txt
+          java -jar ./spdx-tools-2.2.7-jar-with-dependencies.jar Verify sbom.json
```

Comment:
This jar has been deprecated for the [new version of the tools](https://github.com/spdx/tools-java/releases/download/v1.0.4/tools-java-1.0.4.zip). The new URL to download the new jar is:
https://github.com/spdx/tools-java/releases/download/v1.0.4/tools-java-1.0.4.zip

(the version is lower but the jar is newer)

Reply:
thanks!

Reply:
this was a big help, I think it caught a bunch of things we'd missed with the old tool 🙏